src/Main.elm

```diff
@@ -149,6 +169,23 @@ view model =
         ]
 
 
+renderCardSetRadios : String -> List (Html Msg)
+renderCardSetRadios selectedCardSet =
+    List.map
+        (\{ title, file } ->
+            [ input
+                [ type_ "radio"
+                , onClick (SelectedCardSet file)
+                , checked (file == selectedCardSet)
+                ]
+                []
+            , label [] [ text title ]
```

Comment:
label and input need to be programmatically linked, or wrapped so the label can be clicked